### ISSUE TITLE
I don't understand _isSendableEvent

### ISSUE THREAD

**Issue by infojunkie** [chrisguttandin/midi-player#354]
I am trying to capture some MIDI events in my own `IMidiOutput` implementation, so I changed the implementation of `_isSendableEvent` to always return true, regardless of the event.

If I use the same code to send the events to a real Web MIDI output, some calls fail with the following error:
```
Uncaught TypeError: Failed to execute 'send' on 'MIDIOutput': Running status is not allowed at index 1 (6).
```

I looked up "Running status" and I see it is indeed not allowed in the Web MIDI spec for `MIDIOutput.send()` https://www.w3.org/TR/webmidi/#methods-2

So my question is: Does this mean that only the 4 events mentioned in `_isSendableEvent` are allowed by the Web MIDI spec?
```
return 'controlChange' in event || 'noteOff' in event || 'noteOn' in event || 'programChange' in event;
```
or is there a deeper issue, e.g. with `encodeMidiMessage` not outputting a correct sequence for some events and thus confusing the Web MIDI subsystem?

**Comment by chrisguttandin** [comment]
If I remember correctly the reason for `_isSendableEvent()` to allow only these four events is that it was just the requirement for the project I created this library for at the time. It should probably be something that can be configured instead of a private function.

I guess the error message that you got is a false positive. It's probably complaining about a meta event. These events are allowed in MIDI files but as far as I know they are not send over the wire.

Considering that, I would suggest to change `_isSendableEvent()` to allow everything but meta events. It could then be overwritten when creating a player to support more specific use cases. Does that sound like a good plan?

**Comment by infojunkie** [comment]
Thanks for taking the time to answer me.

> It's probably complaining about a meta event.

Yes, that's exactly what was happening.

> I would suggest to change _isSendableEvent() to allow everything but meta events.

Yes, that would be great!

> It should probably be something that can be configured instead of a private function.

I agree! That's what I ended up [doing in my fork](https://github.com/chrisguttandin/midi-player/compare/master...infojunkie:midi-player:infojunkie#diff-6a9b8df2a03ca5533c802f9f61e6eb943bb3e4a5c390734d44c6a183d8979080R30).

**Comment by chrisguttandin** [comment]
I've opted to leave the default logic as is for now to make sure this is not introducing a breaking change. It should be published as v7.1.0 in a few minutes.

If you want to send all status events it should work by passing using this function:

```js
create({
    filterMidiMessage: (event) => 'channel' in event,
    // ...
});
```

Please let me know if it doesn't work as expected.

**Comment by infojunkie** [comment]
Worked perfectly, thank you!
